File: backend-go/internal/modules/system/entity/sys_operation_log.go
package entity

import (
	"drip-admin/backend-go/internal/common"
	"time"
)

type SysOperationLog struct {
	ID             common.Int64String  `gorm:"column:id;primaryKey" json:"id"`
	OperatorID     *common.Int64String `gorm:"column:operator_id" json:"operatorId"`
	OperatorName   *string             `gorm:"column:operator_name" json:"operatorName"`
	Module         string              `gorm:"column:module" json:"module"`
	Action         string              `gorm:"column:action" json:"action"`
	Method         string              `gorm:"column:method" json:"method"`
	Path           string              `gorm:"column:path" json:"path"`
	RequestParams  *string             `gorm:"column:request_params" json:"requestParams"`
	ResponseStatus string              `gorm:"column:response_status" json:"responseStatus"`
	ErrorMessage   *string             `gorm:"column:error_message" json:"errorMessage"`
	CostMs         common.Int64String  `gorm:"column:cost_ms" json:"costMs"`
	CreatedAt      *time.Time          `gorm:"column:created_at" json:"createdAt"`
}

func (SysOperationLog) TableName() string { return "sys_operation_log" }

File: backend-python/app/modules/system/entity/sys_operation_log.py
from datetime import datetime

from sqlalchemy import BigInteger, DateTime, String, Text
from sqlalchemy.orm import Mapped, mapped_column

from app.modules.system.entity.base import Base


class SysOperationLog(Base):
    __tablename__ = "sys_operation_log"

    id: Mapped[int] = mapped_column(BigInteger, primary_key=True)
    operator_id: Mapped[int | None] = mapped_column(BigInteger, nullable=True)
    operator_name: Mapped[str | None] = mapped_column(String(64), nullable=True)
    module: Mapped[str] = mapped_column(String(64))
    action: Mapped[str] = mapped_column(String(64))
    method: Mapped[str] = mapped_column(String(16))
    path: Mapped[str] = mapped_column(String(255))
    request_params: Mapped[str | None] = mapped_column(Text, nullable=True)
    response_status: Mapped[str] = mapped_column(String(16))
    error_message: Mapped[str | None] = mapped_column(Text, nullable=True)
    cost_ms: Mapped[int] = mapped_column(BigInteger)
    created_at: Mapped[datetime] = mapped_column(DateTime)
